### DISCUSSION TITLE
why app name is invalid?

### DISCUSSION THREAD

**Discussion by ITfisher** [loco-rs/loco#435] [Q&A]
<img width="872" alt="image" src="https://github.com/loco-rs/loco/assets/18683222/4c20d902-991e-4f51-8fa2-ce4f815737ff">

**Comment by kaplanelad** [comment]
the name that you wrote with dot? or spaces?

**Reply by ITfisher** [reply]
i am sure i use the name "my_app" ,not use dot or space.and i tried many times.

**Reply by ITfisher** [reply]
but it not work now.

**Reply by ITfisher** [reply]
1.
loco new
. myapp
2.
loco new
myapp
==
i tried tow times.it not work

**Comment by ITfisher** [comment]
i write only name "myapp".
and it shows like that:
<img width="478" alt="image" src="https://github.com/loco-rs/loco/assets/18683222/6878b2c5-2b1c-4381-ac94-b34194eaa928">
what should i do?

**Comment by kaplanelad** [comment]
can you please run with log debug and share with me the logs?
```
RUST_LOG=debug loco new
```

**Reply by ITfisher** [reply]
I don't know what caused the previous failure, but today I successfully created it. Anyway, thank you very much for your help. Also, the Loco project is really great.
